rpc-rs/CHANGELOG.md

```diff
@@ -1,10 +1,42 @@
 # wasmbus-rpc Changelog
 
+## BREAKING CHANGES from 0.8.x to 0.9.0
+
+- provider_main has a new parameter: friendly_name
```

Comment:
Can we add a bit more information here denoting that you can pick anything for your friendly name but we chose do do `httpserver` for `wasmcloud:httpserver`?

Reply:
I figured since this will affect _every_ provider it's worth more context